applications/plugins/SofaImplicitField/components/geometry/bottleField.h

```diff
@@ -0,0 +1,79 @@
+/******************************************************************************
+*                 SOFA, Simulation Open-Framework Architecture                *
+*                    (c) 2006 INRIA, USTL, UJF, CNRS, MGH                     *
+*                                                                             *
+* This program is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU Lesser General Public License as published by    *
+* the Free Software Foundation; either version 2.1 of the License, or (at     *
+* your option) any later version.                                             *
+*                                                                             *
+* This program is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License *
+* for more details.                                                           *
+*                                                                             *
+* You should have received a copy of the GNU Lesser General Public License    *
+* along with this program. If not, see <http://www.gnu.org/licenses/>.        *
+*******************************************************************************
+* Authors: The SOFA Team and external contributors (see Authors.txt)          *
+*                                                                             *
+* Contact information: contact@sofa-framework.org                             *
+******************************************************************************/
+#pragma once
+
+#include "ScalarField.h"
+#include <sofa/defaulttype/Mat.h>
+namespace sofa::component::geometry::_bottleField_
+{
+
+
+using sofa::defaulttype::Vec3d;
+using sofa::defaulttype::Mat3x3;
+
+/**
+ * This component emulates an implicit field shaped by a sphere with a hole made by an ellispsoid. The result may look like some kind of bottle or vase.
+*/
+
+class  SOFA_SOFAIMPLICITFIELD_API bottleField  : public ScalarField
```

Comment:
`bottleField` --> `BottleField`
Change to applied on filenames too :disappointed:

Reply:
Ok, I missed the fact all the components were always starting with a capital letter!